### DISCUSSION TITLE
Invalid previous_tag parameter

### DISCUSSION THREAD

**Discussion by wylzabc** [goreleaser/goreleaser#3273] [Q&A]
why when I push my first tag to github, meet the error "400 Invalid previous_tag parameter"?
<img width="907" alt="image" src="https://user-images.githubusercontent.com/37738057/182162993-332fcef1-e2b7-4ee9-807d-53073548a862.png">

**Comment by caarlos0** [comment]
hmm, I think it was a bug, I recall fixing it at some point... do you know if this still happens?
